sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/util/InputStreamSaver.java

```diff
@@ -19,11 +19,11 @@ public static void saveInputStreamToZipFile(String filename, InputStream inputSt
             }
             inputStream.close();
         } catch (IOException e) {
-            throw new XrayWrapperRuntimeException("Could not save https input stream to zip file", e, XrayWrapperExitCode.IO_ERROR);
+            throw new XrayWrapperException("Could not save https input stream to zip file", XrayWrapperExitCode.IO_ERROR, e);
         }
     }
 
-    public static String readInputStreamAsString(InputStream inputStream) throws XrayWrapperRuntimeException {
+    public static String readInputStreamAsString(InputStream inputStream) throws XrayWrapperException {
```

Comment:
The class is called `InputStreamSaver`. 
But it does provide a zip method and a readInputAsStream

Maybe you should rename it to IoUtil (if methods are kept static) or to `IOHelper` or `IOSupport` if you want to use instances.